docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc

```diff
@@ -1,8 +1,303 @@
+{classanalysis}. It contains the number of occurrences where the analysis
+classified data points correctly with their actual class as well as the number
+of occurrences where it misclassified them.
+
+Let's see two examples of the confusion matrix. The first is a confusion matrix 
+of a binary problem:
+
+image::images/confusion-matrix-binary.jpg[alt="Confusion matrix of a binary problem",width="75%",align="center"]
+
+It is a two by two matrix because there are only two classes (`true` and
+`false`). The matrix shows the proportion of data points that is correctly
+identified as members of a each class and the proportion that is 
+misidentified.
+
+As the number of classes in your {classanalysis} increases, the confusion
+matrix also increases in complexity:
+
+image::images/confusion-matrix-multiclass.jpg[alt="Confusion matrix of a multiclass problem",width="100%",align="center"]
+
+
+The matrix contains the actual labels on the left side while the predicted 
+labels are on the top. The proportion of correct and incorrect predictions is 
+broken down for each class. This enables you to examine how the {classanalysis}
+confused the different classes while it made its predictions.
+
+[discrete]
+[[ml-dfanalytics-class-aucroc]]
+==== Area under the curve of receiver operating characteristic (AUC ROC)
+
+The receiver operating characteristic (ROC) curve is a plot that represents the 
+performance of the classification process at different predicted probability 
+thresholds. It compares the true positive rate for a specific class against the 
+rate of all the other classes combined ("one versus all" strategy) at the 
+different threshold levels to create the curve.
+
+Let's see an example. You have three classes: `A`, `B`, and `C`, you want to 
+calculate AUC ROC for `A`. In this case, the number of correctly classified 
+++A++s (true positives) are compared to the number of ++B++s and ++C++s that are 
+misclassified as ++A++s (false positives).
+
+From this plot, you can compute the area under the curve (AUC) value, which is a 
+number between 0 and 1. The higher the AUC, the better the model is at 
+predicting ++A++s as ++A++s, in this case.
+
+NOTE: To use this evaluation method, you must set `num_top_classes` to `-1`
+or a value greater than or equal to the total number of classes when you create
+the {dfanalytics-job}.
+
+////
+Another crucial measurement is how well your model performs on unseen data
+points. To assess how well the trained model will perform on data it has never
+seen before, you must set aside a proportion of the training data set for 
+testing. This split of the data set is the _testing data set_. Once the model has 
+been trained, you can let the model predict the value of the data points it has 
+never seen before and compare the prediction to the actual value by using the 
+evaluate {dfanalytics} API.
+////
+
+[discrete]
+[[performing-classification]]
+== Performing {classanalysis}
```

Comment:
I think making it clearer this is an example against the sample data is helpful:
```suggestion
== Performing {classanalysis} in the sample flight data set
```